obs-studio-stream-vm-chocolatey/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```

Comment:
remove the parameters you have defaultValues for...

Reply:
@bmoore-msft, I've modified as you suggested, but directly faced the problem: 
"The requested VM size Standard_NV6 is not available in the current region." - I suppose that is related to the subscription which used in CI process. 
If I precise directly in default value the region to use, the template will fail the best practices check. What is the best workaround?

Reply:
I just added that one back, we should probably remove the defaultValue from the location property too, since NV machines aren't in very many regions...  if CI passes, I can make that change when I merge.

Reply:
@bmoore-msft, thank you, CI passed well. I've tested in various officially supported locations for this VM sizes in previous PRs but only few of them is available in CI subscription